### PR TITLE
Image Inpainting GUI modifications

### PR DESCRIPTION
**Pull request by fzhar**
Reset is added

### CODE REVIEW

File: demos/python_demos/image_inpainting_demo/inpainting_gui.py [hunk 1]
```diff
@@ -71,12 +71,17 @@ def run(self):
...
+                self.mask[:, :, :] = 0
+                self.img = self.original_img.copy()
+                self.update_window()
             elif key == ord('\t'):
                 self.is_original_shown = not self.is_original_shown
                 self.update_window()
             elif key in (ord('h'), ord('H')):
                 if not self.is_help_shown:
-                    self.show_info("Use mouse with LMB to paint\nBksp or C to clear\nSpace or Enter to inpaint\nTab to show original image\nEsc or Q to quit")
+                    self.show_info("Use mouse with LMB to paint\nBksp or C to clear current mask\nSpace or Enter to inpaint\nR to reset all changes\nTab to show original image\nEsc or Q to quit")
```
Comment (eizamaliev) [inline demos/python_demos/image_inpainting_demo/inpainting_gui.py:84]: Line too long. Please split it into two or more (according to `\n` splitting)

**Review by eizamaliev — CHANGES_REQUESTED** [review]

File: demos/python_demos/image_inpainting_demo/inpainting_gui.py (hunk 1 above)
Comment (fzhar) [reply demos/python_demos/image_inpainting_demo/inpainting_gui.py]: Done

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**